### PR TITLE
feat: unity realtime docs.

### PR DESCRIPTION
**Pull request by wujun4code**
may the force be with Unity.

### CODE REVIEW

File: views/realtime-unity.md [hunk 1]
```diff
@@ -0,0 +1,886 @@
...
+#### WebSocket 协议
+LeanCloud 实时消息是基于 WebSocket 和私有通讯协议实现的一套聊天系统，因此开发者最好是提前了解一下 WebSocket 协议的相关内容。推荐没有接触过的开发者可以阅读下面这个链接里面的内容来做一个简单的了解[WebSocket 是什么原理？为什么可以实现持久连接？ - 回答作者: Ovear](http://zhihu.com/question/20215561/answer/40316953)
+
+#### Unity 可用的 WebSocket 库
+目前 SDK 内置的 WebSocket 库是一个来自于开源社区的版本 [sta/websocket-sharp](https://github.com/sta/websocket-sharp)，它有个致命的缺陷就是：不支持 iOS 9.0 以上的版本，因此我们推荐开发者可以考虑购买一个付费的版本：[WebSocket for desktop, web and mobile](https://www.assetstore.unity3d.com/cn/#!/content/27658)，如果开发者有自己实现 WebSocket 客户端的能力也可以自己实现，并且 SDK 是支持在初始化的时候指定 WebSocket 客户端的。详细的可以阅读自定义配置中的[自定义 WebSocket 客户端](#自定义_SDK)
+
+## 安装 SDK
+
+### 下载和导入
+开发者需要从：https://github.com/leancloud/realtime-SDK-dotNET/releases 这里下载最新版本的 SDK，并且把里面包含的所有的 Dll 文件都引入到 Unity 的 Assets/LeanCloud 文件夹（在 Assets 下面新建一个 LeanCloud 文件夹用来存放 LeanCloud SDK）下即可。
```
Comment (myleslee) [inline views/realtime-unity.md]: https://github.com/leancloud/realtime-SDK-dotNET/releases
为方便开发者访问，github 上的 SDK 都迁移到了 https://releases.leanapp.cn/#/

File: views/realtime-unity.md [hunk 2]
```diff
@@ -0,0 +1,886 @@
+# Unity 实时通信开发指南（C#）
+
+## 准备工作
+
+### 您需要拥有的开发工具
+
+- Unity 5.3.x 以上的版本
+  这个是因为在 Unity 5.3 之后，官方使用 UnityWebRequest 取代了原有的 WWW 类作为 HTTP 请求的客户端，因此 LeanCloud SDK 在实时通信上是使用了 UnityWebRequest 来获取云端分配的 WebSocket 链接地址。
+- Visual Studio 2015 以上的版本
+微软针对 Unity 有一个工具包可以让 Unity 和 Visual Studio 联调，详细请看[使用 Visual Studio Tools for Unity](https://msdn.microsoft.com/zh-cn/library/dn940020.aspx)，因此建议用户在 Winodows 上开发 Unity 的体验会更好，当然在 Mac OS 以及其他操作系统上开发也没有任何问题。
```
Comment (myleslee) [inline views/realtime-unity.md]: - 开头空两个格
- typo: Winodows

File: views/realtime-unity.md [hunk 3]
```diff
@@ -0,0 +1,886 @@
...
+    {
+        // 构建对话的时候需要指定一个 AVIMClient 实例做关联
+        var TaoYuanConversation = AVIMConversation.CreateWithoutData(TaoYuanConversationId, liubei);
+        // 刘备将徐庶从对话中删除
+        return liubei.KickAsync(TaoYuanConversation, "1005");
+    });
+}
+```
+
+其他在线成员会收到 ``AVIMClient.OnMembersLeft`，而徐庶本人会收到 `AVIMClient.OnKicked`
```
Comment (myleslee) [inline views/realtime-unity.md]: ``AVIMClient.OnMembersLeft`
前面多了一个撇

File: views/realtime-unity.md [hunk 4]
```diff
@@ -0,0 +1,886 @@
...
+    {
+        // 曹操建立与程昱的私聊
+        caocao.CreateConversationAsync(member: "2002", name: "曹操与程昱的私聊");
+    });
+}
+```
+
+上述代码跟刘备创建「桃园」时的区别只有一个，刘备传入的是关羽和张飞两个人的 client Id  作为一个集合参数，曹操这里传入的是程昱这一个人单个的 member 的 client Id ，因此本质上我们依然只能通过成员数量来判断是群聊还是单聊，为了日后更好区分单聊和私聊，我们推荐开发者在创建对话的时候给对话赋予一个自定义的属性，来标记这个对话的类型：
+
+```
```
Comment (myleslee) [inline views/realtime-unity.md]: 加上 cs

Comment (wujun4code) [inline views/realtime-unity.md]: done.

File: views/realtime-unity.md (hunk 3 above)
Comment (wujun4code) [inline views/realtime-unity.md]: done.

File: views/realtime-unity.md (hunk 2 above)
Comment (wujun4code) [inline views/realtime-unity.md]: done.

File: views/realtime-unity.md (hunk 1 above)
Comment (wujun4code) [inline views/realtime-unity.md]: done.

**Comment by myleslee** [comment]
Reviewed

**Pull request merged**